pkg/controller/core.oam.dev/v1alpha2/application/application_controller_test.go

```diff
@@ -1100,9 +1097,8 @@ spec:
     name: deployments.apps
   extension:
     template: |
```

Comment:
please add a new test case instead of modifying the existing tests, we need to make sure that its backward compatible.